hieradata/cluster/antu.yaml

```diff
@@ -0,0 +1,35 @@
+---
+lookup_options:
+  network::interfaces_hash:
+    merge:
+      strategy: "deep"
+  yum::managed_repos:
+    merge:
+      strategy: "unique"
+  yum::repos:
+    merge:
+      strategy: "deep"
+classes:
+  - "network"
+packages:
+  - "perccli"  # from dell repo
+network::interfaces_hash:
+  em1:  # fqdn
+    bootproto: "dhcp"
+    defroute: "yes"
+    onboot: "yes"
+    type: "Ethernet"
+  em2:
+    bootproto: "none"
+    onboot: "no"
+    type: "Ethernet"
+yum::managed_repos:
+  - "dell"
+yum::repos:
```

Comment:
We have a number of settings that we apply for Dell hardware, and this configuration isn't specific to/particularly fundamental for the antu cluster. I'd like to extract a `dell_server` class for details like this - what's your thinking on that approach?

Reply:
The only dell specific resource is the installation of `perccli` and the associated yum repo.  I do not object to a perccli profile being created so long as the **data** stays in yaml.